Index: src/main/java/com/example/skyboxjavafxtester/SkyBoxController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/skyboxjavafxtester/SkyBoxController.java b/src/main/java/com/example/skyboxjavafxtester/SkyBoxController.java
--- a/src/main/java/com/example/skyboxjavafxtester/SkyBoxController.java	
+++ b/src/main/java/com/example/skyboxjavafxtester/SkyBoxController.java	(date 1647205565536)
@@ -5,7 +5,7 @@
 import javafx.event.ActionEvent;
 import javafx.fxml.FXML;
 import javafx.scene.Group;
-
+import javafx.scene.Node;
 import javafx.scene.control.Button;
 import javafx.scene.control.DatePicker;
 import javafx.scene.control.Slider;
@@ -71,7 +71,7 @@
 
     @FXML
     protected void initialize() throws ParseException, IOException {
-        //setSkyboxPane();
+        setSkyboxPane();
         setEntireFrame();
 
         //Initialize Slider Ticks
@@ -93,8 +93,8 @@
 
 
     @FXML
-    protected static Pane setSkyboxPane(Group skyboxGroup) throws ParseException {
-
+    protected Pane setSkyboxPane() throws ParseException {
+        Group skyboxGroup = new Group();
         Group skyBox = SkyBoxApplication.createSkybox(skyboxGroup);
 
         //TODO pretty sure we need to pull in the details of the skybox in createskybox() and pass them to be added to the skybox pane group below.
@@ -106,10 +106,9 @@
         SkyBoxApplication.startParams(); // Setting start date, location, sunset/sunrise times
         Group panelsWHouse = SkyBoxApplication.models(); //Creating all models for the scene
         SkyBoxApplication.constructWorld(skyBox); // Construct the empty SkyBox group
-        skyBox.getChildren().addAll(skyboxGroup, sun, panelsWHouse);
+        skyboxGroup.getChildren().addAll(skyBox, sun, panelsWHouse);
 
         skyboxPane = new Pane(sun, skyboxGroup);
-
         //THIS causes an error: Duplicate children added so i commented it out. Its being out in skyboxPane above
      //   skyboxPane.getChildren().add(skybox);
 
